### PR TITLE
Updated models.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: tests/system/action/agenda_item/test_numbering.py [hunk 1]
```diff
@@ -43,11 +53,11 @@ def test_numbering_without_parents(self) -> None:
         )
         self.create_model(
             "agenda_item/1",
-            {"meeting_id": 1, "weight": 10, "type": 1},
+            {"meeting_id": 1, "weight": 10, "type": "common"},
```
Comment (jsangmeister) [inline tests/system/action/agenda_item/test_numbering.py:56]: Can you use the enum here as well?

File: tests/system/action/agenda_item/test_numbering.py [hunk 2]
```diff
@@ -43,11 +53,11 @@ def test_numbering_without_parents(self) -> None:
...
         )
         self.create_model(
             "agenda_item/2",
-            {"meeting_id": 1, "weight": 10, "type": 1},
+            {"meeting_id": 1, "weight": 10, "type": "common"},
```
Comment (jsangmeister) [inline tests/system/action/agenda_item/test_numbering.py:60]: .

File: tests/system/action/group/test_delete.py [hunk 3]
```diff
@@ -41,13 +41,13 @@ def test_delete_default_group(self) -> None:
 
         self.assert_status_code(response, 400)
 
-    def test_delete_superadmin_group(self) -> None:
+    def test_delete_dmin_group(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/group/test_delete.py:44]: typo

**Comment by normanjaeckel** [comment]
All done, ready to merge.

**Pull request merged**